<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitoringWindow</class>
 <widget class="QMainWindow" name="MonitoringWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Monitoring - Weapon Detection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="infoText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>360</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Configure su sistema de monitoreo ingresando la ubicación de la cámara y un contacto para recibir notificaciones de alertas de seguridad.</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   
   <widget class="QLabel" name="ubicationLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ubicación de la Cámara:</string>
    </property>
   </widget>
   
   <!-- CAMPO DE UBICACIÓN CON ALTURA CORREGIDA -->
   <widget class="QLineEdit" name="ubicationInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>340</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Ej: Oficina Principal - Entrada Norte</string>
    </property>
   </widget>
   
   <widget class="QLabel" name="receiveLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Contacto para Notificaciones:</string>
    </property>
   </widget>
   
   <!-- CAMPO DE CONTACTO CON ALTURA CORREGIDA -->
   <widget class="QLineEdit" name="receiveInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>340</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Ej: +593987654321 o email@empresa.com</string>
    </property>
   </widget>
   
   <!-- BOTONES REPOSICIONADOS -->
   <widget class="QPushButton" name="startMonitoring">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>355</y>
      <width>150</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>▶ Monitorear</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>355</y>
      <width>150</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>✕ Cerrar Sesión</string>
    </property>
   </widget>

   <!-- Área para la barra de progreso REPOSICIONADA -->
   <widget class="QFrame" name="progressFrame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>425</y>
      <width>340</width>
      <height>15</height>
     </rect>
    </property>
   </widget>

  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>